.config/nextest.toml

```diff
@@ -0,0 +1,8 @@
+[profile.ci]
```

Comment:
`cargo insta` doesn't support passing custom test runner flags. Luckily, nextest supports setting the profile via environment flag..